difficulties/Hazard_5.5x2_60_Nitra.cd.json

```diff
@@ -94,6 +104,50 @@
   "EnvironmentalDamageModifier": 1.21,
   "PointExtractionScalar": 1.15,
   "SpeedModifier": 1.2,
+  "EnemyDescriptors": {
+    "ED_FacilityTurret_Barrier": {
+      "Base": "ED_FacilityTurret_Barrier",
+      "DifficultyRating": 24,
+      "IdealSpawnSize": 5,
+      "MaxSpawnCount": 5000,
+      "SpawnAmountModifier": 0.5
+    },
+    "ED_FacilityTurret_Burst": {
+      "Base": "ED_FacilityTurret_Burst",
+      "DifficultyRating": 24,
+      "IdealSpawnSize": 5,
+      "MaxSpawnCount": 5000,
+      "SpawnAmountModifier": 0.5
+    },
+    "ED_FacilityTurret_Sniper": {
+      "Base": "ED_FacilityTurret_Sniper",
+      "DifficultyRating": 24,
+      "IdealSpawnSize": 5,
+      "MaxSpawnCount": 5000,
+      "SpawnAmountModifier": 0.5
+    },
+    "ED_PatrolBot": {
+      "Base": "ED_PatrolBot",
+      "DifficultyRating": 150,
+      "IdealSpawnSize": 5,
+      "MaxSpawnCount": 5000,
+      "SpawnAmountModifier": 0.5
+    },
+    "ED_Shredder": {
+      "Base": "ED_Shredder",
+      "DifficultyRating": 24,
+      "IdealSpawnSize": 17.5,
+      "MaxSpawnCount": 5000,
+      "SpawnAmountModifier": 0.5,
+      "MinSpawnCount": 2
+    },
+    "ED_Terminator": {
+      "Base": "ED_Terminator",
+      "DifficultyRating": 280,
+      "MaxSpawnCount": 1,
+      "SpawnAmountModifier": 0.5
+    }
+  },
```

Comment:
What is the reason behind tweaking descriptors? 5.5 is a vanilla difficulty and vanilla difficulties with a multiplier generally only change enemy count, enemy cap, and resupply cost. A difficulty changing anything else should be named something else.

Reply:
(I have no clue tbh, whoeven sent me this file and asked me to help submit a change)
I should remove these descriptor changes.

Reply:
Removed.